app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -31,41 +31,11 @@ object FeatureFlags {
      */
     const val syncAddressesFeature = false
 
-    /**
-     * Enables the onboarding sync CFR on the home screen.
-     */
-    const val showSynCFR = true
-
-    /**
-     * Enables the onboarding jump back in CFR on the home screen.
-     */
-    const val showJumpBackInCFR = true
-
-    /**
-     * Enables the first run onboarding updates.
-     */
-    const val showFirstRunOnboardingUpdates = false
-
-    /**
-     * Enables the "recent" tabs feature in the home screen.
-     */
-    const val showRecentTabsFeature = true
-
     /**
      * Enables UI features based on history metadata.
      */
     const val historyMetadataUIFeature = true
```

Comment:
Can we remove `historyMetadataUIFeature` too?

Reply:
This was intended as a followup. If you look at the conditional in https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt#1266, it gets kinda weird because we essentially have `true || isHistoryMetadataEnabled`, but losing that `true` from removing `historyMetadataUIFeature` might have some unintended consequences. So, it required digging into `isHistoryMetadataEnabled`, which felt a bit too much in this simple PR.

Reply:
`isHistoryMetadataEnabled` is used only for `historyMetadataUIFeature` so it looks like that is fallback code that wasn't removed before which you could remove here too or just turn `featureFlag` to true for `historyMetadataUIFeature` if you'd like.